docs/gitbook/usage/webhooks.md

```diff
@@ -83,16 +83,19 @@ Webhook payload (HTTP POST):
 
 ```javascript
 {
-    "name": "podinfo",
-    "namespace": "test",
-    "phase": "Progressing", 
-    "metadata": {
-        "test":  "all",
-        "token":  "16688eb5e9f289f1991c"
-    }
+  "name": "podinfo",
+  "namespace": "test",
+  "phase": "Progressing",
+  "checksum": "string (canary hash)",
```

Comment:
Let's put here an actual hash and `"checksum": "string (canary hash)"` should go in the `Event payload` example.

Reply:
I've added these.